docs/dev/graphics-stack.md

```diff
@@ -0,0 +1,398 @@
+        7th | HASWELL D      | Haswell              | 0x0D0B  | CRW GT1 reserved      |              |
+        7th | HASWELL D      | Haswell              | 0x0D1B  | CRW GT2 reserved      |              |
+        7th | HASWELL D      | Haswell              | 0x0D2B  | CRW GT3 reserved      |              |
+        7th | HASWELL D      | Haswell              | 0x0D0E  | CRW GT1 reserved      |              |
+        7th | HASWELL D      | Haswell              | 0x0D1E  | CRW GT2 reserved      |              |
+        7th | HASWELL D      | Haswell              | 0x0D2E  | CRW GT3 reserved      |              |
+        7th | HASWELL M      | Haswell              | 0x0406  | GT1 mobile            |              |
+        7th | HASWELL M      | Haswell              | 0x0416  |GT2 mobile             |              |
+        7th | HASWELL M      | Haswell              | 0x0426  |GT2 mobile             |              |
+        7th | HASWELL M      | Haswell              | 0x0C06  |SDV GT1 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0C16  |SDV GT2 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0C26  |SDV GT3 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0A06  |ULT GT1 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0A16  |ULT GT2 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0A26  |ULT GT3 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0A0E  |ULX GT1 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0A1E  |ULX GT2 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0A2E  |ULT GT3 reserved       |              |
+        7th | HASWELL M      | Haswell              | 0x0D06  |CRW GT1 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0D16  |CRW GT2 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0D26  |CRW GT3 mobile         |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f30|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f31|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f32|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f33|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0157|                      |              |
+        7th | VALLEY VIEW D  | Valley View (Bay Trail) | 0x0155|                      |              |
+        8th | BROADWELL GT12M| Broadwell            | 0x1602  | GT1 ULT               |Gen >=8 **NOT** supported at all.|
```

Comment:
I guess remove gen 8 and newer completely? It does not seem to suit anything.

Reply:
It's useful at least for one purpose - check device id and see that it's completely unsupported :)